introduction.md

```diff
@@ -0,0 +1 @@
+hey - Create a new file called introduction.md introducing yourself to the dojo. Make sure this file contains an introduction, then add, commit and push your change to the fork. other random stuff
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.